src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocstyle/InputJavadocStyle9.java

```diff
@@ -396,4 +396,4 @@ public class SampleTest {
 }</code></pre>
      */
     public void test3() {} // ok
-}
+}
```

Comment:
Revert this change, we should only update check documentation and spellcheck in scope of this issue

Reply:
my bad, thanks.

Reply:
You removed more than required lines hence this error. Instead of manually reverting changes, you can use `git restore` or some in-IDE feature to do the same.

Reply:
@Vyom-Yadav Done. please check the updates.

Reply:
@0xbakry Please resolve https://github.com/checkstyle/checkstyle/pull/12897#discussion_r1154006646 too.

Reply:
Resolved.